app/tests/test_request_interest.py

```diff
@@ -15,7 +15,7 @@ def test_get_all_interests(self):
     def test_retrieve_valid_single_interest(self):
         self.client.credentials(HTTP_AUTHORIZATION="Bearer " + self.token)
         response = self.client.get(f"/interests/{self.request_interest.pk}/")
-        interest = RequestInterest.objects.get(pk=self.request_interest.id)
+        interest = RequestInterest.objects.get(pk=self.request_interest.pk)
```

Comment:
Same here. Why isn't it `RequestInterest.objects.get(id=self.request_interest.id)` Seems inconsistent since in the views you query the RequestInterest object using id,not pk. https://github.com/mentorpaired/core/blob/29a650969889b18e39286151173f3136b2204148/app/views/requests_interests.py#L47